src/platforms/go/common/troubleshooting.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Troubleshooting
+description: "Troubleshooting common Go Issues"
+keywords: ["go", "troubleshooting"]
+sidebar_order: 1000
+supported:
+  - go
+  - go.echo
+  - go.fasthttp
+  - go.gin
+  - go.http
+  - go.iris
+  - go.martini
+  - go.negroni
+---
+
+## I'm using Go and I don't see a stack trace
+
+Since the Go SDK tries to read the source files from your local disk, it could be possible that you’re compiling your binary and deploying it in a manner that makes the source code inaccessible. 
+
+This is really common in Go because you can simply compile the binary, then have no need for the source code anymore. Without access to the source code, we can’t map anything back and unfortunately, we don’t support uploading Go source code with a release to stitch this data on.
```

Comment:
```suggestion
This is common in Go because you can compile the binary, and then have no more need for the source code. Without access to the source code, however, we can’t map anything back, and we don’t support uploading Go source code with a release to stitch this data on.
```